src/app.js

```diff
@@ -1,13 +1,43 @@
-var express = require("express");
+var createError = require('http-errors');
+var express = require('express');
+var path = require('path');
+var cookieParser = require('cookie-parser');
+var logger = require('morgan');
+
+var indexRouter = require('./routes/index');
+var coursesRouter = require('./routes/courses');
+var pagesRouter = require('./routes/pages');
 
 var app = express();
 
-app.get("/", function (req, res) {
-  res.send("Hello World!");
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'jade');
+
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.use('/', indexRouter);
+app.use('/courses', coursesRouter);
+app.use('/pages', pagesRouter);
+
+// catch 404 and forward to error handler
+app.use(function(req, res, next) {
+  next(createError(404));
 });
 
-app.get("/health", function (req, res) {
```

Comment:
we need this route, its used by the deployment pipeline to check if the app was deployed properly